front/src/common/Map/Buttons/ButtonMapKey.tsx

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { BsQuestionCircle } from 'react-icons/bs';
+import Tipped from '../../../applications/editor/components/Tipped';
 
 type ButtonMapKeyProps = {
   toggleMapKey: () => void;
 };
 
-const ButtonMapKey = ({ toggleMapKey }: ButtonMapKeyProps) => (
-  <button
-    type="button"
-    className="btn-rounded btn-rounded-white btn-map-settings"
-    onClick={toggleMapKey}
-  >
-    <span className="sr-only">Key</span>
-    <i className="icons-circle-tooltip" />
-  </button>
-);
+const ButtonMapKey = ({ toggleMapKey }: ButtonMapKeyProps) => {
+  const { t } = useTranslation('translation');
+  return (
+    <Tipped mode="left">
+      <button
+        type="button"
+        className="btn-rounded btn-rounded-white btn-map-settings"
+        onClick={toggleMapKey}
+      >
+        <span className="sr-only">Key</span>
+        <BsQuestionCircle />
+      </button>
+      <span>{t('common.help-display')} </span>
```

Comment:
I think a better translation would be "legend", can you add new translation keys since help-display is already used somewhere else ?
```suggestion
      <span>{t('common.display-legend')} </span>
```